File: Data/Maintenance.cs
namespace Data
{
	public class Maintenance
	{
		public int Id { get; set; }

		public string Name { get; set; }

		public double Coefficient { get; set; }

		public int SortOrder { get; set; }


		public int FlatId { get; set; }

		public virtual Flat Flat { get; set; }


		public int MaintenanceTypeId { get; set; }

		public virtual MaintenanceType MaintenanceType { get; set; }


		public int? MaintenanceTarifId { get; set; }

		public virtual MaintenanceTarif MaintenanceTarif { get; set; }
	}
}

File: Comunal/app/models/maintenance.ts
export class Maintenance {
    Id: number;
    Selected: boolean;
    Tarif: number | null = null;
    Counters: any;
    State: string;

    constructor(
        public Name: string = null,
        public FlatId: number = 0,
        public MaintenanceTypeId: number = 0,
        public SortOrder: boolean = true,
        public Coefficient: number = 1,
        public MaintenanceTarifId: number | null = null
        ) { }
}